tool/tctl/common/plugin/okta.go

```diff
@@ -172,13 +177,14 @@ func (p *PluginsCommand) InstallOkta(ctx context.Context, args installPluginArgs
 	settings := &types.PluginOktaSettings{
 		OrgUrl: oktaSettings.org.String(),
 		SyncSettings: &types.PluginOktaSyncSettings{
-			SsoConnectorId:  oktaSettings.samlConnector,
-			AppId:           oktaSettings.appID,
-			SyncUsers:       oktaSettings.userSync,
-			SyncAccessLists: oktaSettings.accessListSync,
-			DefaultOwners:   oktaSettings.defaultOwners,
-			GroupFilters:    oktaSettings.groupFilters,
-			AppFilters:      oktaSettings.appFilters,
+			SsoConnectorId:           oktaSettings.samlConnector,
+			AppId:                    oktaSettings.appID,
+			SyncUsers:                oktaSettings.userSync,
+			DisableOktaRequesterRole: oktaSettings.userSync && !oktaSettings.assignOktaRequesterRole,
```

Comment:
What if there is not user sync but there is SCIM enabled ?

Reply:
Thanks, I didn't think about SCIM-only.

Reply:
https://github.com/gravitational/teleport/compare/72b2ce3a3e9742da6909e6c30bbdf5740f4869b2..a9e9610050862be8fd02b607d61962743efe994c